mainapp/models.py

```diff
@@ -1,40 +1,24 @@
 from django.db import models
-from django.contrib.auth.models import User
-from django.db.models.signals import post_save
-from django.dispatch import receiver
+from django.contrib.auth.models import AbstractUser
 from phonenumber_field.modelfields import PhoneNumberField
 
 
-class Profile(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-
+class SiteUser(AbstractUser):
+    full_name = models.CharField(max_length=255, default="")
     phone_number = PhoneNumberField(default="")
     current_location = models.CharField(max_length=255, default="")
     work_location = models.CharField(max_length=255, default="")
 
-    user_type_string_map = {"T": "Tenant", "R": "Renter", "L": "Landlord"}
-
+    user_type_string_map = {"T": "Tenant", "R": "Renter", "L": "Landlord"}  # noqa F821
     user_type = models.CharField(
         max_length=2, choices=user_type_string_map.items(), default="R"
     )
 
     timestamp = models.DateTimeField(auto_now_add=True, auto_now=False)
     updated = models.DateTimeField(auto_now_add=False, auto_now=True)
 
-    def user_type_string(self):
-        return Profile.user_type_string_map.get(self.user_type, "")
+    def user_type_string(self):  # noqa F821
```

Comment:
What is F821? I guess it's suppressing flake8 issues?

Reply:
Yeah this was to suppress a flake8 issue.  Interestingly, whatever the issue was (can't remember) it is gone now. If i remove the F821 suppress, flake8 is no longer complaining.

Reply:
Good! It would be great to double check other noqa markers are necessary!